docs/guides/mParticle-migration-guide.md

```diff
@@ -0,0 +1,299 @@
+    </td>
+    <td>
+    ```swift
+    Amplitude.instance().setUserId("abc")
+    Amplitude.instance().identify(
+      AMPIdentify()
+        .set("top_region", value: "Europe")
+    )
+    ```
+    </td>
+    </tr>
+    <tr>
+    <td> Track </td>
+    <td>
+    ```swift
+    if let event = MPEvent(name: "Video Watched", type: MPEventType.navigation) {
+    event.customAttributes = ["category": "Destination Intro", "title": "Paris"]
+    MParticle.sharedInstance().logEvent(event)
+    }
+    ``` 
+    </td>
+    <td>
+    ```swift
+    Amplitude.instance().logEvent("Video Watched", withEventProperties: ["category": "Destination Intro", "title": "Paris"] )
+    ``` 
+    </td>
+    </tr>
+    <tr>
+    <td> Group </td>
+    <td>
+    </td>
+    <td>
+    ```swift
+    Amplitude.instance().setGroup("orgName", groupName:NSString(string:"xyz"))
+    Amplitude.instance().groupIdentifyWithGroupType(
+      "orgName",
+      groupName:NSString(string:"xyz"),
+      groupIdentify:AMPIdentify().set("plan", value: "enterprise")
+    )
+    ``` 
+    </td>
+    </tr>
+    </table>
+
+
+=== "Android"
+
+    Documentation for [Android Kotlin SDK](/data/sdks/android-kotlin/).
+
+    <table>
+    <tr>
+    <td>  </td> <td> <b>mParticle</b> </td> <td> <b>Amplitude</b> </td>
+    </tr>
+    <tr>
+    <td> Identify </td>
+    <td> 
+    ```kotlin
+    IdentityApiRequest modifyRequest = IdentityApiRequest.withEmptyUser()
+        .email("updated-email@example.com")
+        .build()
+
+    MParticle.getInstance().Identity().modify(modifyRequest)
+    ```
+    </td>
+    <td>
+    ```kotlin
+    amplitude.setUserId("abc")
+    amplitude.identify(Identify().set("email", "updated-email@example.com"))
+    ```
+    </td>
+    </tr>
+    <tr>
+    <td> Track </td>
+    <td>
+    ```kotlin
+    val customAttributes: MutableMap<String, String> = HashMap()
+    customAttributes["name"] = "Moto 360"
+    val event = MPEvent.Builder("Product Viewed", MParticle.EventType.Navigation)
+                .customAttributes(customAttributes)
+                .build()
+    MParticle.getInstance()?.logEvent(event)
+    ``` 
+    </td>
+    <td>
+    ```kotlin
+    amplitude.track(
+      "Product Viewed",
+      mutableMapOf<String, Any?>("name" to "Moto 360")
+    )
+    ``` 
+    </td>
+    </tr>
+    <tr>
+    <td> Group </td>
+    <td>
+    </td>
+    <td>
+    ```kotlin
+    amplitude.setGroup("orgName", "xyz");
+    amplitude.groupIdentify("orgName", "xyz", Identify().set("plan", "enterprise"))
+    ``` 
+    </td>
+    </tr>
+    </table>
+ 
+For all other SDKs view the [Quickstart Guide](/data/sdks/sdk-quickstart/) and take a look at the [SDK documentation](/data/sdks/sdk-overview/).
+
+## Validate events
+
+Data validation is a critical step in the instrumentation process. Amplitude lets validate your event data via Amplitude's [User Lookup](/data/debugger/#user-lookup) or using the [Instrumentation Explorer](/data/debugger/#instrumentation-explorer) Chrome extension.
```

Comment:
Maybe also add a link to this for a full list of look-up tools https://www.docs.developers.amplitude.com/data/debugger/